lib/pages/infoPages/vendorPage.dart

```diff
@@ -255,101 +415,145 @@ class _VendorButtonRowState extends State<VendorButtonRow> {
   void initialize() async {
     user = await _auth.currentUser();
```

Comment:
should we have try block in case of a potential error when trying to retrieve the current user?

Reply:
Does auth throw? I can check.

Reply:
I get it now. All awaits can throw because of the future. This has been changed in the latest commit.